### PR TITLE
[KOGITO-8401] Add a brief introduction to Kogito SW capabilities in the home page

### PR DESCRIPTION
**Pull request by ricardozanini**
Signed-off-by: Ricardo Zanini <zanini@redhat.com>

**JIRA:** https://issues.redhat.com/browse/KOGITO-8401

**Description:** Brief introduction explaining the potential usage of Kogito SW in the cards.

Please make sure that your PR meets the following requirements:

- [x] You have read the [contributions doc](https://github.com/kiegroup/kogito-docs/blob/main/CONTRIBUTING.md)
- [x] Pull Request title is properly formatted: `KOGITO-XYZ Subject`
- [x] Pull Request title contains the target branch if not targeting main: `[0.9.x] KOGITO-XYZ Subject`
- [x] The nav.adoc file has a link to this guide in the proper category
- [x] The index.adoc file has a card to this guide in the proper category, with a meaningful description

<details>
<summary>
How to backport a pull request to a different branch?
</summary>

In order to automatically create a **backporting pull request** please add one or more labels having the following format `backport-<branch-name>`, where `<branch-name>` is the name of the branch where the pull request must be backported to (e.g., `backport-7.67.x` to backport the original PR to the `7.67.x` branch).

> **NOTE**: **backporting** is an action aiming to move a change (usually a commit) from a branch (usually the main one) to another one, which is generally referring to a still maintained release branch. Keeping it simple: it is about to move a specific change or a set of them from one branch to another.

Once the original pull request is successfully merged, the automated action will create one backporting pull request per each label (with the previous format) that has been added.

If something goes wrong, the author will be notified and at this point a manual backporting is needed.

> **NOTE**: this automated backporting is triggered whenever a pull request on `main` branch is labeled or closed, but both conditions must be satisfied to get the new PR created.
</details>

### CODE REVIEW

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 1]
```diff
@@ -1,5 +1,13 @@
 = {page-component-title}
 
+{product_name} is a tool to build cloud-native workflow applications. {product_name} can be used to do services and events orchestration and choreografy. Currently, with {product_name} you can integrate with services and events in your architecture using:
```
Comment (spolti) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:3]: what choreografy means in this context?

Comment (ricardozanini) [reply serverlessworkflow/modules/ROOT/pages/index.adoc:3]: Here: https://en.wikipedia.org/wiki/Service_choreography

Comment (spolti) [reply serverlessworkflow/modules/ROOT/pages/index.adoc:3]: I see, thanks.

**Review by spolti — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/index.adoc (hunk 1 above)
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:3]: We can avoid the passive voice. 
```suggestion
{product_name} is a tool to build cloud-native workflow applications. You can use the {product_name} to do the services and events orchestration and choreography. Currently, with {product_name} you can integrate with services and events in your architecture using:
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 2]
```diff
@@ -1,5 +1,13 @@
...
+
+1. **CloudEvents**. Ideal for an Event-Driven architecture where the services are ready to consume and produce events working in more reactive way.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:5]:
```suggestion
1. **CloudEvents**. Ideal for an Event-Driven architecture where the services are ready to consume and produce events working in a more reactive way.
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 3]
```diff
@@ -1,5 +1,13 @@
...
+2. **Sync or Async REST services invocations via OpenAPI/Async API**. There are options also to directly call a REST service in the architecture or ecosystem. Either async and sync methods are supported depending on your requirements.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:6]:
```suggestion
2. **Sync or Async REST services invocations via OpenAPI/Async API**. There are options even to directly call a REST service in the architecture or ecosystem. Either async or sync methods are supported depending on your requirements.
```

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 4]
```diff
@@ -1,5 +1,13 @@
...
+3. **Internal Service execution or invocation**. {product_name} is also a framework to build Java Workflow applications. You can use it to create custom Java services to run an application process within the same instance.
+
+Learn how to do start creating, managing, and deploying your workflow applications with the guides below.
```
Comment (kaldesai) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:9]:
```suggestion
You can learn how to create, manage, and deploy your workflow applications with the following guides.
```

**Review by kaldesai — APPROVED** [review]
@ricardozanini I have added my suggestions. Otherwise, your content looks good to me. Thank you!

**Review by radtriste — COMMENTED** [review]
Don't we want `{product_name} {context}` instead of `{product_name}`
we are using only a part of Kogito.

**Comment by ricardozanini** [comment]
> Don't we want `{product_name} {context}` instead of `{product_name}` we are using only a part of Kogito.

The problem is that `product_name` and `context` are "OpenShift Serverless Logic" downstream. @kaldesai wdyt? I believe we need to review those.

**Review by lazarotti — APPROVED** [review]

File: serverlessworkflow/modules/ROOT/pages/index.adoc [hunk 5]
```diff
@@ -1,5 +1,13 @@
 = {page-component-title}
 
+{product_name} is a tool for building cloud-native workflow applications. You can use the {product_name} to do the services and events orchestration and choreography. Currently, with {product_name} you can integrate with services and events in your architecture using:
```
Comment (MarianMacik) [inline serverlessworkflow/modules/ROOT/pages/index.adoc:3]: I agree with @radtriste, here we should probably use `{product_name} {context}` as we are referring to a specific part of Kogito.

**Review by MarianMacik — APPROVED** [review]
Looks OK, just a minor comment. I think we should closely specify which part of Kogito we talk about.

**Comment by ricardozanini** [comment]
@MarianMacik @radtriste do you mind taking a final look? Can merge if it's fine.

**Review by radtriste — APPROVED** [review]

**Pull request merged**